moztelemetry/heka/message_parser.py

```diff
@@ -35,7 +38,10 @@ def _parse_heka_record(record):
             # because it contains null values for JSON fields that have been
             # split out.
             if field.name == 'submission':
-                payload = _parse_json(field.value_bytes[0].decode('utf-8'))
+                try:
+                    payload = _parse_json(field.value_bytes[0].decode('utf-8'))
```

Comment:
It appears simply calling `.decode('utf-8', 'ignore')` may be the better solution. The test case to reproduce in the bug works and this would map more closely to what we're doing in scala.